doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -175,6 +175,7 @@ nrf9160 samples
         AT command termination methods can be configured using Kconfig options.
         The default method, is CR termination.
```

Comment:
```suggestion
        The default method is CR termination.
```
Sorry, this part was not really subject to review, but I happened to see an extra comma here. I missed it myself in the previous review.